docs/pages/upgrading/agent-managed-updates/bootstrap-ec2-agent.mdx

```diff
@@ -0,0 +1,249 @@
+---
+title: Bootstrapping EC2 agents with "teleport-update"
+description: Describes how to set up EC2 instance by cloud-init script and install Teleport with "teleport-update enable"
+labels:
+ - conceptual
+ - platform-wide
+---
+
+This guide shows two cloud-init patterns for bringing Linux EC2 instances under Managed Updates v2
+and assigning them to an update group using:
+
+```code
+sudo teleport-update enable -g <group_name> -p <proxy>
+```
+
+It assumes you'll join the agent to your cluster during first boot (token or delegated join method)
+and that your cluster has Managed Updates v2 enabled.
+
+What `teleport-update enable` responsible for:
+ - Installs Teleport to the cluster-advertised version and turns on Managed Updates v2 on the host.
+ - Creates systemd units for teleport and a teleport-update timer that periodically runs `teleport-update update`.
+ - Persists most flags you pass (e.g., -g, -p, -b) so re-running enable updates the saved settings.
+
+For full Managed Updates v2 instructions, see [Managed Updates forAgents (v2)](./agent-managed-updates.mdx).
+
+Groups & schedules: Every agent belongs to an update group (e.g., development, staging, prod).
+The cluster-side `autoupdate_config` resource defines when each group may update.
+Cloud or self-hosted clusters use the same schedule model, self-hosted clusters also set the desired
+version via `autoupdate_version`.
+
+## Define groups and schedules
+
+Pick simple, meaningful groups like `development`, `staging`, `production`. Then model your rollout windows
+and sequencing in `autoupdate_config`.
+
+```yaml
```

Comment:
This guide is telling the user to figure out which values to enter based on the example. I would be more descriptive about the precise fields and value types the user should enter.